src/test/ccl/ut_cclut_mock_table.inc

```diff
@@ -594,6 +594,89 @@ subroutine test_cclutRemoveMockTable_created_table(null)
+            call cclutAssertvcEqual(CURREF, "test_cclutRemoveMockTable_created_table_group1 007", repType, "REC")
+            call cclutAssertvcEqual(CURREF, "test_cclutRemoveMockTable_created_table_group1 008",
+                cclut_drop_table_reply->status_data.status, "S")
+            set mockTdbexecuteWasCalled = 1
+            return(0)
+    end
+
+    call cclutAddMockImplementation("tdbexecute", "mock_cclutremovemocktable_created_table_group1_tdbexecute")
+    call cclutExecuteProgramWithMocks("ut_cclut_removeMockTable_grp1", "")
+
+    call cclutAsserti4Equal(CURREF, "test_cclutRemoveMockTable_created_table_group1 009", mockTdbexecuteWasCalled, 1)
+end ;test_cclutRemoveMockTable_created_table_group1
+
+/* test_cclutRemoveMockTable_created_table_group1_error *****************************************************
+*  Scenario: Generates an error when attempting to drop the CCL definition as a group1 user                 *
+************************************************************************************************************/
+subroutine test_cclutRemoveMockTable_created_table_group1_error(null)
+    free record cclut_drop_table_reply
+    record cclut_drop_table_reply (
+%i cclsource:status_block.inc
+    ) with protect
+    set cclut_drop_table_reply->status_data.status = "S"
+
+    declare mockTdbexecuteWasCalled = i4 with protect, noconstant(0)
+    declare mock_cclutRemoveMockTable_created_table_group1_error_tdbexecute(appNum = f8, taskNum = f8, reqNum = f8,
+        reqType = vc, req = vc(ref), repType = vc, rep = vc(ref)) = i4 with protect
+    subroutine mock_cclutRemoveMockTable_created_table_group1_error_tdbexecute(appNum, taskNum, reqNum, reqType, req,
+        repType, rep)
+            call cclutAsserti4Equal(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 001", appNum, 3070000)
+            call cclutAsserti4Equal(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 002", taskNum,
+                3070001)
+            call cclutAsserti4Equal(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 003", reqNum, 3050002)
+            call cclutAssertvcEqual(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 004", reqType, "REC")
+            call cclutAssertvcEqual(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 005",
+                cclut_drop_table_request->program_name, "cclut_drop_table_definition")
+            call cclutAssertStartsWith(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 006",
+                concat("^MINE^,^CUST_CCLUT"), cclut_drop_table_request->params)
+            call cclutAssertvcEqual(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 007", repType, "REC")
+            call cclutAssertvcEqual(CURREF, "test_cclutRemoveMockTable_created_table_group1_error 008",
+                cclut_drop_table_reply->status_data.status, "S")
+            set mockTdbexecuteWasCalled = 1
+            return(1)
```

Comment:
This tests for a crm perform error. It is possible for the tdbexecute call to succeed but for something bad to happen inside of cclut_drop_mock_table_definition suggesting there should yet another test.